packages/react-native/ReactAndroid/src/main/java/com/facebook/react/uimanager/TransformHelper.java

```diff
@@ -130,6 +152,14 @@ public static void processTransform(
     }
   }
 
+  private static double parsePercentageToValue(String stringValue, double dimension) {
+    if (stringValue.endsWith("%")) {
```

Comment:
I'm not sure the old code was reliable here, but we usually want to avoid native crashes on invalid style values. So we probably want to be handling `NumberFormatException` somewhere.

Reply:
makes sense, updated!